### PR TITLE
Metadata update

### PR DESCRIPTION
**Pull request by Rplesha**
I think we have worked out all of the alias differences. Things are working on the website with the final few changes in this branch.

The pd_all_alias files should be accepted from my branch when we merge if needed.

### CODE REVIEW

File: utils/data/target_metadata/julia_alias_file.csv [hunk 1]
```diff
@@ -333,7 +333,7 @@ IC1613 GHV 61331,GHV2009-Star-61331,[GHV2009] Star 61331,IC1613-61331,IC1613-010
 IC1613 GHV 67559,GHV2009-Star-67559,[GHV2009] Star 67559,IC1613-67559,IC1613-010505-020923,,,,,,
 IC1613 BUG B7,BUG2007-B7,[BUG2007] B 7,[GHV2009] Star 63932,GAIA DR3 2539060283142934656,IC1613-010502-020805,,,,,
 IC1613 BUG A10 ,BUG2007-A10,[BUG2007] A 10,[GHV2009] Star 62390,[SK76] A22,GAIA DR3 2539073687736158848,IC1613-010501-021041,,,,
-IC1613 BUG B4,BUG2007-B4,[BUG2007] B 4,[GHV2009] Star 60449,[SK76] A42,GAIA DR3 2539072107187468672,IC1613-010460-020916,,,,
+IC1613 BUG B4,BUG2007-NB4,[BUG2007] B 4,[GHV2009] Star 60449,[SK76] A42,GAIA DR3 2539072107187468672,IC1613-010460-020916,,,,
```
[the remark this answers is not in the document]
Comment (jduval82) [reply utils/data/target_metadata/julia_alias_file.csv:336]: @Rplesha yes, should just be B4

[the remark this answers is not in the document]
Comment (Rplesha) [reply utils/data/target_metadata/julia_alias_file.csv]: Fixed!

**Comment by Rplesha** [comment]
@jotaylor I'd appreciate your look over the code changes I made here, too before we merge it!

**Review by jotaylor — APPROVED** [review]
The changes look good and the updated pd_all_aliases file looks as expected 👍

**Pull request merged**